Sources/Onboarding/Styles/DaxDialogStyles.swift

```diff
@@ -30,49 +30,73 @@ extension OnboardingStyles {
             configuration.label
                 .font(.system(size: fontSize, weight: .bold))
                 .fixedSize(horizontal: false, vertical: true)
                 .multilineTextAlignment(.center)
                 .lineLimit(nil)
-                .foregroundColor(foregroundColor(configuration.isPressed))
+                .foregroundColor(foregroundColor(isPressed: configuration.isPressed, isHovered: isHovered))
                 .padding()
-                .frame(minWidth: 0, maxWidth: .infinity, maxHeight: maxHeight)
-                .background(backgroundColor(configuration.isPressed))
+                .frame(minWidth: 0, maxWidth: maxWidth, maxHeight: maxHeight)
+                .background(backgroundColor(isPressed: configuration.isPressed, isHovered: isHovered))
                 .cornerRadius(8)
                 .contentShape(Rectangle()) // Makes whole button area tappable, when there's no background
+                .onHover { hovering in
+                    #if os(macOS)
+                    self.isHovered = hovering
+                    #endif
+                }
         }
 
-        private func foregroundColor(_ isPressed: Bool) -> Color {
-            switch (colorScheme, isPressed) {
-            case (.dark, false):
-                return .blue30
-            case (.dark, true):
-                return .blue20
-            case (_, false):
-                return .blueBase
-            case (_, true):
-                return .blue70
+        private func foregroundColor(isPressed: Bool, isHovered: Bool) -> Color {
+            switch (colorScheme, isPressed, isHovered) {
+            case (.light, false, false):
+                return .lightRestBlue
+            case (.dark, false, false):
+                return .darkRestBlue
+            case (.light, false, true):
+                return .lightHoverBlue
+            case (.dark, false, true):
+                return .darkHoverBlue
+            case (.light, true, _):
+                return .lightPressedBlue
+            case (.dark, true, _):
+                return .darkPressedBlue
+            case (_, _, _):
+                return .lightRestBlue
             }
         }
 
-        private func backgroundColor(_ isPressed: Bool) -> Color {
-            switch (colorScheme, isPressed) {
-            case (.light, true):
-                return .blueBase.opacity(0.2)
-            case (.dark, true):
-                return .blue30.opacity(0.2)
-            default:
+        private func backgroundColor(isPressed: Bool, isHovered: Bool) -> Color {
+            switch (colorScheme, isPressed, isHovered) {
+            case (.light, false, false):
+                return .black.opacity(0.01)
+            case (.dark, false, false):
+                return .white.opacity(0.03)
+            case (.light, false, true):
+                return .black.opacity(0.03)
+            case (.dark, false, true):
```

Comment:
nip: the cases that return the same value could go on same line but I’m fine with this if it reads better.